docs/quick-start.MD

```diff
@@ -257,6 +257,28 @@ spec:
 EOF
 ```
 
+ For connecting `VMAgent` to cluster, you have to specify url for it (not a path: `/insert/0/prometheus/api/v1/write`):
```

Comment:
This message can confuse user, please describe the correct param but not error

Example
The full URL is required for VMAgent as remoteWrite URL.  Example    remoteWrite:
     - url: "http://vminsert-example-vmcluster.default.svc.cluster.local:8480/insert/0/prometheus/api/v1/write"